x/oracle/keeper/msg_server_create_price.go

```diff
@@ -2,22 +2,30 @@ package keeper
 
 import (
 	"context"
+	"errors"
 	"strconv"
+	"time"
 
 	"github.com/ExocoreNetwork/exocore/x/oracle/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+const layout = "2006-01-02 15:04:05"
```

Comment:
can we move this const timelayout into oralce/types ? 
the latter one variable  t also use this format ? 

>` const t = "2024-05-01 01:01:01"`

why not use one ?

Reply:
That is required by golang. The layout is a special sequence that Go uses to understand how to interpret and format date and time strings. Random string will cause error.